pkg/util/util.go

```diff
@@ -125,3 +125,14 @@ func GetEsHostname(opts map[string]string) string {
 	urlArr := strings.Split(urls, ",")
 	return urlArr[0]
 }
+
+// HasArg returns whether the slice has an entry prefixed by the given name
+func HasArg(name string, args []string) bool {
+	for _, v := range args {
+		if strings.HasPrefix(v, name) {
```

Comment:
Why `HasPrefix` rather than equals? The method `HasArg` implies checking for a particular argument, not whether an arg with a prefix.

Reply:
Perhaps the name needs to have `=` suffix just to make sure it matches the full arg name

Reply:
> Why HasPrefix rather than equals?

we don't know what user specified as the value

Reply:
Yeah realised when looking at the items passed - hence suggesting we should add `=` to the supplied name, just to confirm we are locating the full argument name.

Reply:
Done